res/values/cm_strings.xml

```diff
@@ -28,6 +39,24 @@
     <!-- Default value for the default snooze delay (in minutes) -->
     <string name="preferences_default_snooze_delay_default" translatable="false">5</string>
 
+    <!-- Delete events -->
+    <string name="events_delete">Delete events</string>
+    <string name="action_select_all">Select all</string>
+    <string name="action_select_none">Deselect all</string>
+    <string name="action_delete">Done</string>
+    <string name="all_events">All events</string>
+    <string name="no_events">No events available</string>
+    <string name="evt_del_dlg_title">Delete events?</string>
+    <string name="evt_del_dlg_msg_selected">Selected events will be deleted</string>
+    <string name="evt_del_dlg_msg_all">All events will be deleted</string>
+    <string name="no_events_selected">No events selected</string>
+
+  <plurals name="events_selected">
+    <item quantity="zero">No events selected</item>
```

Comment:
Do you know how to select events? I wasn't able to do it. And the string only seems to be used here.